code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -778,23 +782,23 @@
 						user.visible_message("<span class='warning'>A [rite.name] appears at [user]'s feet!</span>", \
 							 "<span class='cultitalic'>A [rite.name] materializes at your feet.</span>")
 			if("Blood Bolt Barrage (300)")
```

Comment:
```suggestion
			if("Blood Bolt Barrage ([BLOOD_BARRAGE_COST])")
```

Reply:
List options do not allow variables, only static strings it seems.

Reply:
Planning on moving costs to the tooltip with /examine() in a seperate PR